docs/data/base/guides/working-with-tailwind-css/working-with-tailwind-css.md

```diff
@@ -0,0 +1,503 @@
+           <svg width="30" height="32" fill="currentColor">
+             <rect x="6" y="4" width="4" height="24" rx="2" />
+             <rect x="20" y="4" width="4" height="24" rx="2" />
+           </svg>
+-        </button>
++        </Button>
+         <div className="flex-auto flex items-center justify-evenly">
+-          <button type="button" aria-label="Skip 10 seconds">
++          <Button aria-label="Skip 10 seconds">
+             <svg width="24" height="24" fill="none">
+               <path d="M17.509 16.95c-2.862 2.733-7.501 2.733-10.363 0-2.861-2.734-2.861-7.166 0-9.9 2.862-2.733 7.501-2.733 10.363 0 .38.365.711.759.991 1.176" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M19 5v3.111c0 .491-.398.889-.889.889H15" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="hidden sm:block lg:hidden xl:block" aria-label="Next">
++          </Button>
++          <Button className="hidden sm:block lg:hidden xl:block" aria-label="Next">
+             <svg width="24" height="24" fill="none">
+               <path d="M14 12 6 6v12l8-6Z" fill="currentColor" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M18 6v12" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="rounded-lg text-xs leading-6 font-semibold px-2 ring-2 ring-inset ring-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500">
++          </Button>
++          <Button className="rounded-lg text-xs leading-6 font-semibold px-2 border-2 border-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500 hover:ring-cyan-500">
+             1x
+-          </button>
++          </Button>
+         </div>
+       </div>
+     </div>
+```
+
+Some classes were slightly changed on some buttons so we have a consistent focus indicator.
+
+## What we learned
+
+These are the things we covered in this guide:
+
+✅ How to use Tailwind CSS utility classes to style MUI Base components, using the `componentsProps` prop for targeting specific slots within the component.\
+✅ How to create custom components for specific slots in more complex customization scenarios.
+We used the `component` prop to pass them into the parent component.\
+✅ How to apply conditional styling based on the owner component's state using a callback as value for the `componentsProps` prop.
+
+Get all the code used in this guide in the [MUI Base with Tailwind CSS](https://github.com/mui/material-ui/tree/master/examples/mui-base-with-tailwind-css) example project.
```

Comment:
In the page, https://github.com/mui/material-ui/tree/master/examples/mui-base-with-tailwind-css I believe that we miss the generic steps as we use for the other example, e.g. https://github.com/mui/material-ui/tree/master/examples/create-react-app. there are links to run locally, in code sandbox, and in stackblitz.